### ISSUE TITLE
Investigate if jQuery Globalize should be leveraged

### ISSUE THREAD

**Issue by nschonni** [wet-boew/wet-boew#6726]
Globalize has now hit 1.0 https://github.com/jquery/globalize so it might be work looking at to see if some of the approaches and library can replace our custom i18n handling.

Looking just at [.loadMessages( json )](https://github.com/jquery/globalize/blob/acbd30c93924ad9883da3ec94ea86567efd22b98/doc/api/message/load-messages.md) it would require a refactor of the message handling since it loads all messages at once, but it would also mean the translations could be loaded on a plugin-by-plugin basis.

**Comment by hsrudnicki** [comment]
@nschonni : For example, here are data access resources, see on http://cldr.unicode.org/index/cldr-spec/json,  http://cldr.unicode.org/translation, which other big companies are building blocks for software to support the world’s languages.
For leveraging the official Unicode CLDR JSON data, a JavaScript library can be used for internationalization and localization, for more details how jQuery Globalize leverages the Common Locale Data Repository, see https://github.com/jquery/globalize, http://blog.jquery.com/2015/04/23/announcing-globalize-1-0/, and others work (from IBM and MS): https://developer.ibm.com/opentech/jquery/ and http://archive.news.softpedia.com/news/Microsoft-Open-Source-jQuery-Globalization-Plugin-144481.shtml

**Comment by hsrudnicki** [comment]
https://github.com/wet-boew/wet-boew/issues/2074 as further to this issue and in regards to jquery UI [see on http://blog.jquery.com/category/jquery-ui/ and http://stackoverflow.com/questions/15700648/is-jquery-mobile-mobile-first]

**Comment by LaurentGoderre** [comment]
jQuery UI and jQuery Mobile are irrelevant to this discussion

**Comment by hsrudnicki** [comment]
@LaurentGoderre : you are right, in greater detail, the both issues can be in two different categories of jQuery as a whole, thank you for your comment

**Comment by hsrudnicki** [comment]
Just by occasion, here is a story about jQuery, CLDR JSON from Globalize project lead: http://rafael.xavier.blog.br/2014/12/23/a-story-of-jquery-cldr-json-and-the-community-at-the-unicode-santa-clara-ca/

**Comment by hsrudnicki** [comment]
Additionally, when applicable, for comparing/checking, see on https://github.com/rxaviers/javascript-globalization

**Comment by hsrudnicki** [comment]
@nschonni @LaurentGoderre : Are you ready to close this issue or otherwise?

**Comment by RobJohnston** [comment]
The use of jQuery Globalize was mentioned in one other issue: https://github.com/wet-boew/wet-boew/issues/6742.

**Comment by hsrudnicki** [comment]
Please continue the update of this issue's status whether this issue can be closed or otherwise? Thx

**Comment by LaurentGoderre** [comment]
@nschonni do you still like this library or would you prefer https://github.com/i18next/i18next? I am looking at integrating this in my prototype
